endoreg_db/logger_conf.py

```
@@ -0,0 +1,101 @@
+import logging
```

Ruff (0.8.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)